TacTok/extract_proof_steps.py

```diff
@@ -107,13 +107,17 @@ def process_proof(filename, proof_data):
 
 if __name__ == '__main__':
     arg_parser = argparse.ArgumentParser(description='Extract the proof steps from CoqGym for trainig ASTactic via supervised learning')
+    arg_parser.add_argument('--toggle_defs', action='store_false', help='toggle whether to include the names of definitions and theorems in the model')
+    arg_parser.add_argument('--toggle_locals', action='store_false', help='toggle whether to include the names of local variables in the model')
```

Comment:
The name "toggle" is confusing to me here, makes it seem like applying this option multiple times will toggle it on and off. How about "no_defs" and "no_locals", and then make the `dest` "use_defs" and "use_locals".

Reply:
So the full command for the first would be:
```
arg_parser.add_argument('--no_defs', action='store_false', dest='use_defs', help="...")
```